runelite-plugin.properties

```diff
@@ -1,5 +1,5 @@
 displayName=Easy Giant's Foundry
-author=Toofifty
+author=Toofifty and Patrick
```

Comment:
Sorry, not too keen on updating the author here. Could we instead add a contributors list to the readme?